smithy-cli/src/test/resources/software/amazon/smithy/cli/commands/projection-model-import.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "1.0",
+  "projections": {
+    "exampleProjection": {
+      "imports": ["unknown-trait.smithy"]
```

Comment:
This is existing file - https://github.com/awslabs/smithy/blob/main/smithy-cli/src/test/resources/software/amazon/smithy/cli/commands/unknown-trait.smithy